brushmodes.c

```diff
@@ -54,7 +54,7 @@ void draw_dab_pixels_BlendMode_Normal (uint16_t * mask,
     for (; mask[0]; mask++, rgba+=4) {
       uint32_t opa_a = mask[0]*(uint32_t)opacity/(1<<15); // topAlpha
       uint32_t opa_b = (1<<15)-opa_a; // bottomAlpha
-      rgba[3] = opa_a + opa_b * rgba[3] / (1<<15);
+      rgba[3] = (opa_a + opa_b * rgba[3]) >> 15;
```

Comment:
:-1:

* Old is `opa_a + ((opa_b * rgba[3]) / (1<<15))` because division and multiplication have a higher precedence than addition and subtraction.
* New is `(opa_a + (opa_b * rgba[3])) / (1<<15)`.